spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe ForgotPasswordsController do
+  describe "POST create" do
+    context "with valid email address" do
+      it "generates a token for the user" do
+        bob = Fabricate :user, email: "bob@bob.com"
+        expect(bob.token).to be_nil
```

Comment:
This practice is not the best - having double expectations are certainly confusing sometimes. I would recommend sticking to single assertion principle. If you want to be sure that the token is there after post then set it nil prior post and assert the present after.